strats.md

```diff
@@ -1114,3 +1115,34 @@ In a heated room, it instead has the form:
 ```
 
 The implicit unlock strats can be disabled by setting the node property `useImplicitDoorUnlocks` to false.
+
+## Collects Items
+
+An `collectsItems` array lists the node IDs (within the room) of items that become collected (if not already collected) as part of executing this strat. This can be useful, for example, for strats that use G-mode to "remote acquire" items at a location different from the item node.
```

Comment:
```suggestion
A `collectsItems` array lists the node IDs (within the room) of items that become collected (if not already collected) as part of executing this strat. This can be useful, for example, for strats that use G-mode to "remote acquire" items at a location different from the item node.
```